src/lib/import_export/csv_parser.cpp

```diff
@@ -198,14 +215,12 @@ size_t CsvParser::_parse_into_chunk(std::string_view csv_chunk, const std::vecto
 }
 
 void CsvParser::_sanitize_field(std::string& field) {
-  const std::string linebreak(1, _meta.config.delimiter);
-  const std::string escaped_linebreak =
-      std::string(1, _meta.config.delimiter_escape) + std::string(1, _meta.config.delimiter);
+  constexpr size_t ESCAPED_LINEBREAK_SIZE = 2;
```

Comment:
Mmmh, setting this as a constant makes it look like a developer-chosen constant, not like something that is dependent on `_escaped_linebreak.size()`. At least add a comment explaining its relation to `escaped_linebreak.size()` and why it always has to be 2.

Reply:
Using `escaped_linebreak.size()` again as it didn't make a difference anyway